addon/components/frost-text.js

```diff
@@ -101,7 +101,7 @@ export default Component.extend(FrostEventsProxyMixin, {
       yield timeout(200) // Duration of the visibility animation
     }
     this.set('isClearEnabled', showClear)
-  }).restartable(),
+  }).enqueue(),
```

Comment:
Your PR description says

> The _clear task causes a focus and a trigger input event which calls the _showClear task twice. 

It would be best to correct the `_clear` task so that it does cause the `_showClear` task to be initiated twice.  This PR is fixing the symptom, not the root cause.

Reply:
I just investigated a little bit further and found that it's not just the `_clear` task that causes `_showClear` twice. When the clear button is clicked, a `focusOut` event occurs before the click event. This is the first time that `_showClear` will be called. The click event then happens which calls `_clear`. `_clear` will cause `_showClear` to be triggered twice. We can make it trigger once by removing the [`.focus()` call here](https://github.com/ciena-frost/ember-frost-core/blob/master/addon/components/frost-text.js#L80) since it's not actually needed to clear the field. However, this still means that `_showClear` will be called for a total of 2 times from the initial `focusOut` event and from `_clear`. I'm not sure if there is a solution to prevent it from being called twice.